code/development-env/config/manifests/minikube-proxy.yml

```diff
@@ -46,19 +46,20 @@ spec:
 ---
```

Comment:
I'm surprised that there aren't two paths in this IngressRule as presumably the web application needs to exist on the `/` route. I would have thought minikube would reject the second rule as the domain is taken... but I might be wrong

Reply:
I wasn't sure it would work either but it does fine, initially I did this as I thought we'd need a re-write annotation for the API but actually we don't, so I'll merge these two into a single ingress rule.